zerver/context_processors.py

```diff
@@ -101,10 +101,13 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
         realm_uri = settings.ROOT_DOMAIN_URI
         realm_name = None
         realm_icon = None
+        realm_plan_type = None
```

Comment:
Here we'd need to decide which message is the default in this case for a boolean value.

You're version had the Zulip Cloud version as the default for the boolean. I would go with the self hosted version here and verify that with Alya when she's back next week.

Reply:
Ok, going with self hosted for now